client/view/workbook/settings/settings.js

```diff
@@ -0,0 +1,16 @@
+import settings from './settings.html'
+
+export default mediator => ({
+	name: `settings`,
```

Comment:
Now that I think about it a bit, why not name this state `info` while we're at it, since I'm imagining that eventually this view will be the place for us to show the user things like how many of the cards they've memorized so far, and what percentage of the cards they've memorized per sheet, and that kind of thing.  More than just settings.